### DISCUSSION TITLE
Help, I'm using the latest version of the lib, but Google is still complaining about the minimum version of the SDK to payment

### DISCUSSION THREAD

**Discussion by MarcosSarges** [hyochan/react-native-iap#2819] [Q&A]
Help, I'm using the latest version of the lib, but Google is still complaining about the minimum version of the SDK to payment.

<img width="735" alt="Captura de Tela 2024-08-14 às 15 28 22" src="https://github.com/user-attachments/assets/a07d768f-7b39-4b3b-bcbc-cf4ddad9e6cb">

<img width="392" alt="Captura de Tela 2024-08-14 às 15 30 22" src="https://github.com/user-attachments/assets/97e18fc7-3116-46ae-a49b-c9688bb569eb">

**Comment by hyochan** [comment]
You're using an **outdated and unsupported version**.

v12/v13 uses older Google Play Billing versions that may no longer meet Google's requirements.

**Solution**: Upgrade to v14+

- [Migration Guide (v13 to v14)](https://hyochan.github.io/react-native-iap/docs/guides/migration-v13-to-v14)
